content/docs/command-reference/version.md

```diff
@@ -14,18 +14,16 @@ usage: dvc version [-h] [-q | -v]
 
 | Line                                        | Detail                                                                                                                                                                |
 | ------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash if running a development version)                                                                                        |
-| `Python version`                            | Version of Python used in the environment where DVC is initialized                                                                                                    |
-| `Platform`                                  | Information about the operating system of the machine                                                                                                                 |
-| [`Binary`](#what-we-mean-by-binary)         | Shows whether DVC was installed from a package or from a binary release                                                                                               |
-| `Package`                                   | Name of the package manager used to install DVC if any (`pip`, `conda`, etc)                                                                                          |
-| `Supported remotes`                         | Remote types that have all needed dependencies installed for                                                                                                          |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the <abbr>workspace</abbr> and the <abbr>cache</abbr> |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the cache and <abbr>workspace</abbr> directories                                                   |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version) and name of the package manager (`pip`, `conda`, etc.) or binary used to install DVC   |
```

Comment:
Should the parenthesis be at the end?

Reply:
and `,` before "and name of the package" pls

Reply:
> Should the parenthesis be at the end

Moved it to the end. But I think it's better suited after "Version of DVC" only.

Reply:
Thanks. The first half of this sentence is easier to read without interrupting it with the parenthesis. And "etc." in this case includes things like `deb` or `exe`, which are binaries 🙂. Resolving